CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [1.0.0-nullsafety.0]
```

Comment:
As null safety is in flutter stable, I think the "-nullsafety.0" suffix is not necessary anymore

Reply:
fixed

Reply:
@victorbonnet